library/block/block_entity/src/testmod/java/org/quiltmc/qsl/block/entity/test/client/mixin/BlockColorsMixin.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.block.entity.test.client.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import net.minecraft.client.color.block.BlockColors;
+
+import org.quiltmc.qsl.block.entity.test.BlockEntityTypeTest;
+import org.quiltmc.qsl.block.entity.test.client.BlockEntityTypeTestClient;
+
+@Mixin(BlockColors.class)
+public class BlockColorsMixin {
+	@Inject(method = "create", at = @At("RETURN"))
+	private static void onCreate(CallbackInfoReturnable<BlockColors> cir) {
+		// @TODO use QSL color provider API instead once available
```

Comment:
```suggestion
		// TODO: use QSL color provider API instead once available
```
It might be better to move to the `TODO:` (or `TODO`) "standard"; VSCode thinks that `@TODO` is a person, which is not good:
![image](https://user-images.githubusercontent.com/85590273/174502309-0f79f74f-7ad0-4ac6-a58d-b7c77de16d79.png)

Reply:
Damn it VSCode.